sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md

```diff
@@ -1,13 +1,12 @@
 # Release History
 
-## 1.0.0-beta.1 (Unreleased)
+## 1.0.0-beta.1 (2023-08-08)
 
 ### Features Added
 
-Initial Release.
+Initial release.
+Supported scenarios: Sending events and receiving events.
 
-### Breaking Changes
+For more information, see README.md.
```

Comment:
@antkmsft would you mind fixing the formatting/spacing for this section. Adding a suggestion becomes tricky here.

Reply:
What are the formatting/spacing errors you would like to see fixed?

Reply:
I updated it directly in this PR as a commit, @LarryOsterman. Added a bullet (for consistency) and removed the extra new lines.

Please take a look and if you prefer something changed, let me know.

Also, based on suggestion from @antkmsft offline, I removed the "refer to the README.md" sentence. It doesn't have a link and might be confusing for folks not reading it on GitHub.

Reply:
The readme file is present in the repo. It should be linked in the changelog.

Reply:
Added the refer to README.md file back with a link to the file for those folks not reading it on GitHub.

Reply:
Changelogs is also available via the Azure SDK blog (https://devblogs.microsoft.com/azure-sdk/azure-sdk-release-july-2023/), specifically https://azure.github.io/azure-sdk/releases/2023-07/cpp.html. Over there, it is not clear which README.md we are talking about, there is no hyperlink. And if you're browsing in this very repo, and get to CHANGELOG.md, it is likely that you already saw the README.md, it is even one directory level above the place where CHANGELOG.md lives (and it gets displayed automatically once you are in the eventhubs directory).

Reply:
As I see it, there are two options here:
1) The changelog.md is where customers discover the eventhubs service and want to discover what functionality is available. In that case, the customer needs to know where to go to find out the rest of the details on how to use the service.
2) The readme.md is where customers discover the eventhubs service, in which case the changelog.md can just have text indicating that it is the initial release.

If the blog strips hyperlinks from the changelog, then that is a bug in the logic which creates the blog post and it should be fixed by the team creating the blog post - if there's a hyperlink in a changelog, it's likely to be important.

Reply:
It's good now - it has the hyperlink (whether this one or the one that Ahson suggests - I'd leave it up to you).
My comment was a bit confusing, sorry - note the 2 minute difference between Larry's commit that adds hyperlink and his comment, and my comment. Basically, I started writing my comment explaining why we want the link before I saw any of these changes. I think we're good now (but having no mentions of README would've been good for me as well).